### ISSUE TITLE
Issue - Sidebar Follow Request Icon Missing

### ISSUE THREAD

**Issue by daw50n** [blobcatz/pebble-mastodon-theme#18]
When user gets a new follower request the sidebar adds a menu for this, with any theme activated an icon is not shown for the sidebar menu entry

<img width="274" alt="Screenshot 2023-11-08 at 08 39 42" src="https://github.com/blobcatz/pebble-mastodon-theme/assets/149823293/f6b9b943-7d81-4ec0-aeb9-d0daaa21ada4">

**Comment by daw50n** [comment]
Agree fix is now present, but when followe requests in the sidebar are selected is the purple colour in keeping with other themed icons?

![Screenshot from 2023-11-08 20-56-03](https://github.com/blobcatz/pebble-mastodon-theme/assets/149823293/4ca68b88-9f3a-4a85-ba33-f636a2c70caf)

![Screenshot from 2023-11-08 20-56-35](https://github.com/blobcatz/pebble-mastodon-theme/assets/149823293/3a8d9bc7-69cb-4e1b-8118-fde31fd42a13)

**Comment by blobcatz** [comment]
My idea was for bookmarks and favorites to have the same color as they are both post-related features and display posts you liked/bookmarked. That's also why the icon for "Explore" is different too. And it's not really "keeping with other themed icons" since "Private mentions", "Live feeds", and "Notifications" also have different colors, the only exception is the "Home" icon.
TLDR: It's a design choice
